src/main.rs

```diff
@@ -156,6 +156,9 @@ Then try to restart Plume
         println!("Please refer to the documentation to see how to configure it.");
     }
 
+    let custom_domains = plume_models::blogs::Blog::list_custom_domains(&dbpool.get().unwrap()).unwrap();
+    dbg!(custom_domains);
```

Comment:
Although it's very useful for debugging and fine for now, I'm gonna point this just to make sure we don't forget it when time for merging come